k9mail/src/main/java/com/fsck/k9/view/MessageHeader.java

```diff
@@ -154,10 +185,31 @@ private void onAddSenderToContacts() {
         }
     }
 
+    private void onAddAddressesToClipboard(Address[] addresses){
+        if(mMessage != null){
```

Comment:
This check is unnecessary because `mMessage` is not used inside the method. Also, by the time we get here `mMessage` will have been dereferenced in all code paths that lead here.